### PR TITLE
Add katello client packages from katello-packaging

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: katello/katello-host-tools/katello-host-tools.spec [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+# crontab
+mkdir -p %{buildroot}%{_sysconfdir}/cron.d/
+cp extra/katello-agent-send.cron %{buildroot}%{_sysconfdir}/cron.d/%{name}
+%endif
+
+%clean
+rm -rf %{buildroot}
+
+%post -n katello-agent
+chkconfig goferd on
```
Comment (ekohl) [inline katello/katello-host-tools/katello-host-tools.spec]: This should be in the EL <= 6 section and ```systemctl enable goferd``` in the systemd section.

File: katello/katello-host-tools/katello-host-tools.spec [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+%postun -n katello-agent
+touch /tmp/katello-agent-restart
+exit 0
+
+%files -n katello-agent
+%defattr(-,root,root,-)
+%config %{_sysconfdir}/gofer/plugins/katelloplugin.conf
+%{_prefix}/lib/gofer/plugins/katelloplugin.*
+
+%doc LICENSE
```
Comment (ekohl) [inline katello/katello-host-tools/katello-host-tools.spec]: Should this be ```%license``` or will that break on older versions?

File: katello/katello-host-tools/katello-host-tools.spec [hunk 3]
```diff
@@ -0,0 +1,376 @@
+Name: katello-host-tools
```
Comment (ekohl) [inline katello/katello-host-tools/katello-host-tools.spec]: I assume all the code to support older versions like EL5 is still relevant in this case.

Comment (ehelms) [reply katello/katello-host-tools/katello-host-tools.spec]: Yup.

File: katello/katello-host-tools/katello-host-tools.spec (hunk 2 above)
Comment (ehelms) [reply katello/katello-host-tools/katello-host-tools.spec]: I think it would break RHEL6 and RHEL5.

**Comment by ehelms** [comment]
Updated

**Pull request merged**